constants/license_classes.json

```diff
@@ -225,5 +225,10 @@
         "All",
         "1",
         "0"
+    ],
+    "LMSYS-Chat-1M Dataset License":[
```

Comment:
@rmahari

Reply:
I think it's All uses, no share alike, no attribution.